package.json

```diff
@@ -49,6 +49,7 @@
     "@uirouter/rx": "^1.0.0"
   },
   "devDependencies": {
+    "@angular-devkit/build-angular": "^14.1.0",
```

Comment:
Might want to set this to version `^14.0.0` so it is consistent with the other versions. I think the same is needed for the `@angular/core` dependency too.

Reply:
sure, updated this